sqf-value/value.h

```diff
@@ -31,6 +31,8 @@
 #include <variant>
 #include <initializer_list>
 
+#undef assert
```

Comment:
Should be placed in main.cpp instead

this (may) cause issues with people including this in their extension and use assert afterwards themself

Reply:
Oops, wrong file yeah 🙈